changelogs/unreleased/3829-unclear-error-when-compiler-venv-has-incompatible-python-version.yml

```diff
@@ -0,0 +1,6 @@
+description: An exception is now raised when there is a mismatch between the python versions of the compiler venv and the active process
+change-type: patch
+destination-branches: [master]
```

Comment:
```suggestion
destination-branches: [master, iso5]
```

Reply:
Should it go to iso4 as well as suggested above by @arnaudsjs ?

Reply:
I think so (see my response there)